src/modules/ShortcutGuide/ShortcutGuide/shortcut_guide.cpp

```diff
@@ -399,6 +409,22 @@ ShortcutGuideSettings OverlayWindow::GetSettings() noexcept
     {
     }
 
+    try
+    {
+        settings.shouldReactToPressedWinKey = (int)properties.GetNamedObject(ShouldReactToPressedWinKey::name).GetNamedBoolean(L"value");
```

Comment:
No need for `(int)` cast

Reply:
Good eye. It's a boolean already.
That's what I get for using copy paste :)